TestFoundation/TestNSURL.swift

```diff
@@ -343,4 +344,11 @@ class TestNSURL : XCTestCase {
         */
     }
 
+    func test_description() {
+        // test with file that exists
+        let path = TestNSURL.gFileExistsPath
+        let url = NSURL(fileURLWithPath: path)
+        
+        XCTAssertEqual(path, url.description, "expected description does not match")
```

Comment:
on the darwin version of Foundation NSURLs come out for their description as "relativeString -- baseURL" if the base url is set, else they are presented as "relativeString -- relativeString"